lib/proxy_api/template.rb

```
@@ -0,0 +1,24 @@
+module ProxyAPI
+  class Template < ProxyAPI::Resource
+    def initialize args
+      @url     = args[:url] + "/unattended"
+      super args
+    end
+
+    # returns the Template URL for this proxy
+    def template_url
+      if (response = parse(get("templateServer"))) and response["templateServer"].present?
+        return response["templateServer"]
+      end
+      false
+    rescue
```

@eLobato first rescue was a typo..left it in by accident.
would you re-raise as a foreman exception or just let it throw as it is.

on the second rescue would you re-raise as foreman exception or just log the error?